cpp/server/handler.cc

```diff
@@ -3,160 +3,77 @@
 #include <algorithm>
 #include <event2/buffer.h>
```

Comment:
Dinged those two (on a related note, we should run IWYU over the codebase rather than trying to do this manually.)